passport-beam/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for passport-beam 1.0.4
+// Project: https://github.com/alfw/passport-beam
+// Definitions by: AtlasDev <https://github.com/AtlasDev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="../passport/passport.d.ts"/>
```

Comment:
This is required. See the guide: http://definitelytyped.org/guides/contributing.html#content